helix-front/client/app/resource/shared/resource.service.ts

```diff
@@ -109,4 +110,16 @@ export class ResourceService extends HelixService {
   public remove(clusterName: string, resourceName: string) {
     return this.delete(`/clusters/${clusterName}/resources/${resourceName}`);
   }
+
+  public setIdealState(
+    clusterName: string,
+    resourceName: string,
+    idealState: Node
+  ) {
+    return this.post(
+      `/clusters/${clusterName}/resources/${resourceName}/idealState?command=update`,
+      JSON.stringify(idealState)
+    );
+  }
```

Comment:
This is the new method to make the network call to https://github.com/apache/helix/blob/master/helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ResourceAccessor.java#L476